server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -488,9 +488,17 @@ private TableMgmtStats manageTablets(Iterator<TabletManagement> iter,
       if (mtiError != null) {
         // An error happened on the TabletServer in the TabletManagementIterator
         // when trying to process this extent.
+        KeyExtent ke = null;
+        try {
+          ke = tm.getExtent();
+        } catch (IllegalStateException e) {
+          // thrown when no prev endrow in tablet metadata.
+          // suppress this exception and leave ke set to null
+        }
+        String errorLocation = ke == null ? " table: " + tm.getTableId() : " extent: " + ke;
```

Comment:
On the server side there is now code in TabletManagementIter that handles when TabletMetadata.convertRow throws an exception.  For that case where happens here on the client side in the manger, will tm be null?  Wondering about the call to `tm.getTableId()` for that case.

Reply:
In TabletManagementIterator.filter, if there are actions or an error, then the keys for the tablet metadata are returned. Before returning the keys, it inserts an action or error key into the set of keys. I don't think tm will be null here.